### ISSUE TITLE
Missing implementation details for the add-widgets modal

### ISSUE THREAD

**Issue by viddo** [CartoDB/cartodb#6635]
This is a meta issue to keep track of the remaining work on the add-widgets modal, since this wasn't planned in the scope of #6565.

Each item should probably be branched out to a separate issue when started to be worked on, defining the actual work and details necessary.
- Inside the modal modal
  - [x] time-series widget should be treated differently #6654
  - [ ] CSS for each widget type's options listings + cards #6682 
  - [ ] do not show options for already added widgets (i.e. all attributes matching a widget, column name+type is not sufficient!)
  - [ ] formula type; generate an option model for each operation (avg, sum, min, max), and add a sub-view (custom tabpane?) to filter per operation type.
  - [ ] render the amount of selections for each type, in the tab pane menu item (e.g. Histogram (3))
  - [ ] show selected options summary in the footer
  - [ ] Show meta info+graphics for each column option (different for each type)
  - [ ] CSS for the tab pane menu
- In the widgets listing
  - [ ] render "add"(-widgets) button next to ([layers, element, widgets]) tab pane, the "add" functionality should depend on current tab-pane selection.  + CSS (smaller size)
  - [ ] zero-case: brief info about widgets + add-button (hide add-button next to parent tab-pane)
- Other
  - [ ] Apply "open sans" font instead of lato (talk w/ @urbanphes before starting this one)

Needs discussion:
- [ ] when selected several widget candidates, one of them could fail for whatever reasons (network issue, 5xx error in API etc.), how should we handle this?

**Comment by xavijam** [comment]
There has been no activity on this issue for more several months. We are closing it. If you think this still needs to be addressed please open a new issue.
